datahub-web-react/src/app/browse/BrowseResultCard.tsx

```diff
@@ -2,6 +2,22 @@ import React from 'react';
 import { Card, Row, Space, Typography } from 'antd';
 import { Link } from 'react-router-dom';
 import { ArrowRightOutlined, FolderOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+
+const styles = {
+    row: { padding: 8 },
+    title: { margin: 0 },
+};
```

Comment:
could you try to move this into a styled component? I'd like to keep at least new styles being added to just `styled`